intro.tex

```diff
@@ -1,55 +1,67 @@
 
-The \esp is motivated by the fact that in the current baseline there will be no science-ready data products released to the community before \drone.
-This is due to the fact that the templates are needed for Difference Imaging are produced in Data Release Production (DRP) and initially released as part of \drone. 
-Alerts cannot be issued until a template image for a given field and filter is available, hence  in the current baseline, no alerts can be issued until after \drone.
+A key component of the \esp is the generation of templates from both the commissioning data and data collected during the first year of the survey.
+In full survey operations, template images for difference image analysis and alert generation are constructed as part of the annual DRP.
+In order to support alert generation in year 1, Rubin will generate templates from all science-grade data taken during commissioning to provide an initial template library at the start of the 10-year survey, and the incrementally generate templates during year 1 using the best images available and covering as much sky in as many filters as possible.
+Details of the current strategy for alert generation  with incremental templates are given in  \S~\ref{sec:pp}.
 
-The success of \es then depends on various scenarios coming out of commissioning as we transition into operations, as described in \S~\ref{ssec:scenarios}. 
 
 \subsection{Early Science scenarios } \label{ssec:scenarios}
 
 Recent planning on the construction project has led to a reduced amount of on-sky time in commissioning, including a reduction in the time dedicated to final science validation of the as-built system compared to earlier draft plans.
-The total amount science validation time is currently planned for 8 weeks.
-As Rubin construction moves through the challenging phase of System Integration, Test and Commissioning (SIT-COM), on--sky time could be further reduced. 
-The Operations team is thus planning for various outcomes that might require special attention to producing \es opportunities in the first part of regular operations to ensure the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky leading to DR1.
+The total amount science validation time currently planned during commissioning is 8 weeks. 
+As Rubin construction moves through the challenging phase of System Integration, Test and Commissioning (SIT-COM), on--sky time could be further reduced.
+
+The Operations team is tracking the progress of the commissioning activities as they relate to \es opportunities to ensure that the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky leading to DR1.
```

Comment:
Consider "prompt" -> "timely" since prompt has a specific meaning in the context of prompt processing.